src/app/fake-cli/fake-cli.fsproj

```diff
@@ -3,7 +3,7 @@
         <PackAsTool>true</PackAsTool>
         <OutputType>Exe</OutputType>
         <ToolCommandName>fake</ToolCommandName>
-        <TargetFramework>net6.0</TargetFramework>
+        <TargetFrameworks>net8.0;net6.0</TargetFrameworks>
```

Comment:
Maybe fake-cli itself should only be 8 regardless of what the modules are? (the integration tests are 8 only as it stands, so if it is still built as 6 as well it moght not get any testing)